src/utils/smartweave/utils.go

```diff
@@ -39,6 +49,18 @@ func ValidateInteraction(tx *arweave.Transaction) (isInteraction bool, err error
 		return false, nil
 	}
 
+	if !inputFromTag {
+		input = tx.Data
+	}
+	// Input must be a valid JSON
+	if jsonError := tool.CheckJSON(input); jsonError != nil {
+		err = fmt.Errorf("value of the input is not a valid JSON: %s", jsonError.Error())
+	}
+
+	if tx.Size() > MaxInteractionDataItemSizeBytes {
+		err = fmt.Errorf("the size of the interaction exceeds the limit: %d bytes", MaxInteractionDataItemSizeBytes)
```

Comment:
we should probably have dedicated error for this and log those interactions differently. But it's a TODO